Makefile

```diff
@@ -208,8 +208,8 @@ EXTERNAL_XA_PATH=$(topdir)
 EXTERNAL_GIT_PATH=$(topdir)/external
 
 prepare-external-git-dependencies:
-	$(call MSBUILD_BINLOG,prep-external-tasks) build-tools/xa-prep-tasks/xa-prep-tasks.csproj
-	$(call MSBUILD_BINLOG,prep-external-checkout) build-tools/xa-prep-tasks/xa-prep-tasks.csproj \
+	msbuild build-tools/xa-prep-tasks/xa-prep-tasks.csproj /p:Configuration=$(CONFIGURATION)
```

Comment:
Any particular reason for this change?  This means we won't get `.binlog` files for the `xa-prep-tasks.csproj` build and `CheckoutExternalGitSources` target invocation.  *Ideally* we shouldn't need such a thing, but when things go "weird", *anything* could be invaluable.

Reply:
This was discussed offline, marking as resolved.